src/ralph/models/xapi/open_response_assessment/fields/objects.py

```diff
@@ -0,0 +1,42 @@
+"""Open Response Assessment xAPI object field definitions"""
+
+from typing import Literal, Optional
+
+from ...base import BaseModelWithConfig
+from ...constants import (
+    ACTIVITY_QUESTION_DISPLAY,
+    ACTIVITY_QUESTION_ID,
+    LANG_EN_US_DISPLAY,
+)
+from ...fields.objects import ObjectDefinitionExtensionsField, ObjectField
+
+
+class QuestionObjectDefinitionField(BaseModelWithConfig):
+    """Represents the `object.definition` xAPI field for question saved xAPI statement.
+
+    WARNING: It doesn't include the recommended `description` field nor the optional `moreInfo`,
+    `Interaction properties` and `extensions` fields.
```

Comment:
Shouldn't we declare them as optional?

Reply:
Good Idea) Updated this part) Now the base `ObjectField` model defines all optional fields and its children redefine some of the fields with additional constraints.